Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -119,7 +119,7 @@ void Jit64::lXXx(UGeckoInstruction inst)
 	// ... maybe the throttle one already do that :p
 	// TODO: We shouldn't use a debug read here.  It should be possible to get
 	// the following instructions out of the JIT state.
-	if (SConfig::GetInstance().bSkipIdle &&
```

Comment:
Yeah, those comments were outdated. We access the next instruction in a safe way "MergeAllowedNextInstructions" "js.op[].inst"